.goreleaser.yml

```diff
@@ -0,0 +1,59 @@
+before:
```

Comment:
Can we include a comment header block with pointers to like the goreleaser docs, etc, for people who want to customize this file further?

Reply:
Done!